rholang/examples/log_time.rho

```diff
@@ -4,9 +4,9 @@
 // and the use of a primitive (log-time) in Rholang.
 
 new time_contract in {
-	contract time_contract(stream) = {
-        log-time(stream)
-	} |
-	time_contract!(ostream-new("test.txt")) |
-	time_contract!(ostream-new("test2.txt"))
+    contract time_contract(stream) = {
+        stream.log-time()
+    } |
+    time_contract!("test.txt".ostream-new()) |
```

Comment:
Ugh. Strings should not have the ambient authority to create files or to write things to the console.  I know we can't fix this properly before the SDK release. @guardbotmk3 What would it take to make an ambient system actor that encapsulates this authority?  Then the tutorial could get them used to the idea of using the `system` contract instead of methods on strings, which are going away.

Reply:
Hmm maybe just disallow primitives (besides print) for the SDK release?

Reply:
As in we just temporarily remove this contract.

Reply:
I didn't see @guardbotmk3 's comment earlier below. I guess I'm just agreeing with him in my comment above.